decode_test.go

```diff
@@ -223,7 +223,7 @@ func TestIssue299HessianDecode(t *testing.T) {
 			"interface_slice": []interface{}{
 				"MyName",
 			},
-			"interface_map": map[string]interface{}{
```

Comment:
unsuport specific key types ??
I have tried Java and it is supported this case

Reply:
currently hessian2 default encode as untyped map, see https://github.com/apache/dubbo-go-hessian2/blob/master/map.go#L122

Reply:
@alpha-baby default encoding as untyped map does not match the hessian2 protocol, u are welcome to fix it if u will

Reply:
> @alpha-baby default encoding as untyped map does not match the hessian2 protocol, u are welcome to fix it if u will

ok, thx